go.mod

```diff
@@ -5,8 +5,8 @@ go 1.23.1
 require (
 	github.com/YakDriver/regexache v0.23.0
 	github.com/hashicorp/boundary v0.17.1
-	github.com/hashicorp/boundary/api v0.0.52
-	github.com/hashicorp/boundary/sdk v0.0.48
+	github.com/hashicorp/boundary/api v0.0.53
```

Comment:
If you want we can update to boundary@main which will pull the current SHA. Then after we release we can make a smaller PR to just update that to 0.18.0, update the changelog date and release the TF provider

Reply:
Sounds good! Updated with https://github.com/hashicorp/terraform-provider-boundary/pull/626/commits/acd3e53e4649bd53773321e87f96a4feb5f9a933